### DISCUSSION TITLE
Issue with Android - overlay not showing

### DISCUSSION THREAD

**Discussion by barbar-binks** [capawesome-team/capacitor-mlkit#121] [Q&A] [answered]
I am a begginer trying to implement a simple QR Code scanning app, everything is working well except that when i launch the scanner with BarcodeScanner.startScan(), I get only the camera, no bounding box or buttons to toggle torch or cancel the scan... I know it seems simple but looking at the docs there is no mention of setting this up in options or adding HTML elements to the scanner. 

let me know! I am following the example, this is my function: 

`
  async startScan() {
    const _self = this;
    document.querySelector('body')?.classList.add('scan-active');
  

    const listener = await BarcodeScanner.addListener(
      'barcodeScanned',
      async result => {
        _self._qrCodes.push(result.barcode);
      },
    );
  
    await BarcodeScanner.startScan({
      formats: [BarcodeFormat.QrCode],
      lensFacing: LensFacing.Back,
    });

  };

`

**Comment by robingenz** [comment, marked as answer]
> I know it seems simple but looking at the docs there is no mention of setting this up in options or adding HTML elements to the scanner.

You can find examples at the end of the [Usage](https://capawesome.io/plugins/mlkit/barcode-scanning/#usage) section. 

Also, feel free to take a look at our working [demo app](https://github.com/robingenz/capacitor-mlkit-plugin-demo).

**Reply by barbar-binks** [reply]
Maybe I have something wrong on my AndroidManifest, but I have triple checked, this is what I have: 

```
<?xml version="1.0" encoding="utf-8" ?>
<manifest xmlns:android="http://schemas.android.com/apk/res/android">
    <application
        android:allowBackup="true"
        android:icon="@mipmap/ic_launcher"
        android:label="@string/app_name"
        android:roundIcon="@mipmap/ic_launcher_round"
        android:supportsRtl="true"
        android:theme="@style/AppTheme">
        <activity
            android:configChanges="orientation|keyboardHidden|keyboard|screenSize|locale|smallestScreenSize|screenLayout|uiMode"
            android:name=".MainActivity"
            android:label="@string/title_activity_main"
            android:theme="@style/AppTheme.NoActionBarLaunch"
            android:launchMode="singleTask"
            android:exported="true">
            <intent-filter>
                <action android:name="android.intent.action.MAIN" />
                <category android:name="android.intent.category.LAUNCHER" />
            </intent-filter>
        </activity>

        <provider
            android:name="androidx.core.content.FileProvider"
            android:authorities="${applicationId}.fileprovider"
            android:exported="false"
            android:grantUriPermissions="true">
            <meta-data android:name="android.support.FILE_PROVIDER_PATHS" android:resource="@xml/file_paths" />
        </provider>

        <meta-data android:name="com.google.mlkit.vision.DEPENDENCIES" android:value="barcode_ui" />

    </application>

    <!-- Permissions -->

    <uses-permission android:name="android.permission.INTERNET" />
    <uses-permission android:name="android.permission.CAMERA" />
    <uses-permission android:name="android.permission.FLASHLIGHT" />
</manifest>

```

and this is what i have on my variables.grade file: 

```
ext {
    minSdkVersion = 22
    compileSdkVersion = 33
    targetSdkVersion = 33
    androidxActivityVersion = '1.7.0'
    androidxAppCompatVersion = '1.6.1'
    androidxCoordinatorLayoutVersion = '1.2.0'
    androidxCoreVersion = '1.10.0'
    androidxFragmentVersion = '1.5.6'
    coreSplashScreenVersion = '1.0.0'
    androidxWebkitVersion = '1.6.1'
    junitVersion = '4.13.2'
    androidxJunitVersion = '1.1.5'
    androidxEspressoCoreVersion = '3.5.1'
    cordovaAndroidVersion = '10.1.1'
    androidxCameraCamera2Version = '1.1.0'
    androidxCameraCoreVersion = '1.1.0'
    androidxCameraLifecycleVersion = '1.1.0'
    androidxCameraViewVersion = '1.1.0'
    mlkitBarcodeScanningVersion = '17.1.0'
    playServicesCodeScannerVersion = '16.0.0'
}

```
> the dependencies starting from androidxCameraCamera2Version are a part of the readme, but i checked the demo app and you don't include them

**Reply by barbar-binks** [reply]
Sorry, finally understood the example, you open a modal with the scan! and the elements are inside the modal I will try that
